src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSpark.java

```diff
@@ -45,129 +51,15 @@
         programGroup=StructuralVariationSparkProgramGroup.class)
 public final class FindBreakpointEvidenceSpark extends GATKSparkTool {
     private static final long serialVersionUID = 1L;
+    private final Logger localLogger = LogManager.getLogger(FindBreakpointEvidenceSpark.class);
```

Comment:
Probably unnecessary for this to be an instance field -- all the methods are static, anyway.  Maybe just call getLogger in runTool?
But I'm fine with the changes to this class.